pkg/test/testutil/testutil.go

```diff
@@ -352,28 +352,13 @@ func writeSpec(dir string, spec *specs.Spec) error {
 	return os.WriteFile(filepath.Join(dir, "config.json"), b, 0755)
 }
 
-// idRandomSrc is a pseudo random generator used to in RandomID.
-var idRandomSrc = rand.New(rand.NewSource(time.Now().UnixNano()))
-
-// idRandomSrcMtx is the mutex protecting idRandomSrc.Read from being used
-// concurrently in different goroutines.
-var idRandomSrcMtx sync.Mutex
-
 // RandomID returns 20 random bytes following the given prefix.
 func RandomID(prefix string) string {
-	// Read 20 random bytes.
 	b := make([]byte, 20)
-	// Rand.Read is not safe for concurrent use. Packetimpact tests can be run in
-	// parallel now, so we have to protect the Read with a mutex. Otherwise we'll
-	// run into name conflicts.
-	// https://golang.org/pkg/math/rand/#Rand.Read
-	idRandomSrcMtx.Lock()
```

Comment:
Are you sure that we don't need this synchronization here?

Reply:
yes, there is a unit test for crypto/rand::Read  at https://github.com/golang/go/blob/c39bc22c141bc6990e4e2abf604dcf56669ff779/src/crypto/rand/rand_test.go#L124